Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -169,7 +171,7 @@
     'questionable',
     'real-estate',
     'recreation-and-hobbies',
-    'reference-and-research	',
+    'reference-and-research    ',
```

Comment:
why this change?

Reply:
Not intentional, looks like it was a tab-to-spaces conversion by my IDE. The whitespace at the end of this string should just be removed, right?

Reply:
please revert

Reply:
@GuyAfik I removed the whitespace because it looks like it was a mistake in the first place – none of the other items in the `PAN_DB_URL_FILTERING_CATEGORIES` array have trailing whitespace.